### PR TITLE
BETA: Register digit.is-a.dev

### PR DESCRIPTION
**Pull request by nt-ossas**
Added `digit.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/digit.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "nt-ossas",
+        "email": "ossas050288@gmail.com"
+    },
+    "record": {
+        "CNAME": "nt-ossas.github.io"
```
Comment (CuteDog5695) [inline domains/digit.json:7]: 404 page (if you have your site hosted then you can comment below a preview of the website)

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/digit.json (hunk 1 above)
Comment (nt-ossas) [reply domains/digit.json:7]: try https://nt-ossas.github.io/DIGIT/techzone/home
or https://nt-ossas.github.io/DIGIT/cryptozone/home

**Comment by nt-ossas** [comment]
File: domains/digit.json
Content URL: nt-ossas.github.io/digit/techzone/home

fix it

File: domains/digit.json (hunk 1 above)
Comment (triach-rold) [reply domains/digit.json:7]: The link you provided works, but you ned to edit that link in your CNAME for it to be considered reachable and complete.

**Review by triach-rold — COMMENTED** [review]
The link you provided works, but you ned to edit that link in your CNAME for it to be considered reachable and complete.

**Comment by nt-ossas** [comment]
why does it look like it doesn’t have the right css or js? should i change any link?

**Pull request closed without merging**